samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2018, Google,LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+
+'use strict';
+
+function localizeObjects(fileName) {
+  // [START vision_localize_objects]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+  const fs = require('fs');
+
+  // Standard Endpoint
+  const client = new vision.ImageAnnotatorClient();
+  const request = {
+    image: {content: fs.readFileSync(fileName)},
+  };
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const fileName = 'Local image file, e.g. /path/to/image.png';
+  client
+    .objectLocalization(request)
+    .then(results => {
+      const objects = results[0].localizedObjectAnnotations;
+      objects.forEach(object => {
+        console.log(`Name: ${object.name}`);
+        console.log(`Confidence: ${object.score}`);
+        console.log(
+          JSON.stringify(object.boundingPoly.normalizedVertices, null, `\t`)
```

Comment:
Please print the `x` and `y` coordinates

```py
print('Number of objects found: {}'.format(len(objects)))
for object_ in objects:
    print('\n{} (confidence: {})'.format(object_.name, object_.score))
    print('Normalized bounding polygon vertices: ')
    for vertex in object_.bounding_poly.normalized_vertices:
        print(' - ({}, {})'.format(vertex.x, vertex.y)) # <– <– <– x and y coordinates
```

As a general rule, I will never ever ever ever ever accept a PR which includes `JSON.stringify`

_*with exceptions where it's appropriate_